File: 1944-truncate-sentence/truncate-sentence.py
class Solution:
    def truncateSentence(self, s: str, k: int) -> str:
        p = 0
        f=""
        for each in s:
            if each == " ":
                p+=1
                if p == k:
                    break
                f+=each
            else:
                f+=each
        return f

File: 1944-truncate-sentence/truncate-sentence.js
/**
 * @param {string} s
 * @param {number} k
 * @return {string}
 */
var truncateSentence = function(s, k) {
    let p = 0;
    let trun = "";
    for (let each of s){
        if (each == " "){
            p++;
            if (p == k)
            break;
            trun += each;
        }
        else
        trun += each;
    }
    return trun;
};